content/pda.md

```diff
@@ -0,0 +1,523 @@
+# PDAs
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
+
+- Explain Program Derived Addresses (PDAs)
+- Explain various use cases of PDAs
+- Describe how PDAs are derived
+- Use PDA derivations to locate and retrieve data
+
+# TL;DR
+
+- A Program Derived Address (PDA) is derived from a program ID and an optional list of seeds.
+- PDAs are designed to be controlled by the program they are derived from
+- PDAs provide a deterministic way to find data stored on accounts
+- Programs can sign instruction for PDAs derived from its program ID
+
+## What is a Program Derived Address
+
+Program Derived Addresses (PDAs) are effectively account addresses designed to be signed for by a program rather than a secret key. As the name suggests, PDAs are derived using a program ID and an optional set of seeds. A PDA has the same form as a Solana public key, expect they are ensured to not be on the ed25519 curve and thus have no associated private key.
+
+PDAs serve two main functions:
+
+1. Provide a deterministic way to find the address of an account
+2. Authorize a program to sign for a PDA in the same way a user may sign with the private key associated with their public key.
```

Comment:
@ZYJLiu I feel like the "associated with..." is redundant?

ALSO, I feel like "on behalf of" is less ambiguous than "for"

> 2. Authorize a program to sign on their behalf in the same way a user may sign with their private key.